components/public-api/gitpod/experimental/v1/idp.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
Nit, I'd name the file in full - identity-provider. Just because it avoids an extra mental lookup